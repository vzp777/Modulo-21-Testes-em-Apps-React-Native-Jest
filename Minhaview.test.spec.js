import React from "react";
import renderer from 'react-test-renderer' 
import MinhaView from "./Minhaview.spec";
import { CliqueAqui } from "./Minhaview.spec";


test('verifica renderização', () => {
    const tree = renderer.create(<MinhaView />).toJSON()
    expect(tree).toMatchSnapshot()

    
});

test('verifica cliqueAqui', () => {
    expect (CliqueAqui()).toBeTruthy()

});