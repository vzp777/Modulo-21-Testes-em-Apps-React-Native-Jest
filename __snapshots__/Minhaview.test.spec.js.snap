// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`verifica renderização 1`] = `
<view>
  <TextInput
    value="Meu Texto"
  />
  <View
    accessibilityRole="button"
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        [
          {},
        ]
      }
    >
      <Text
        style={
          [
            {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Clique Aqui
      </Text>
    </View>
  </View>
</view>
`;
